gradle/build-docs.gradle

```diff
@@ -9,211 +9,78 @@ buildscript {
-		new File("$buildDir/asciidoc/guide/FullGuide-docinfo.xml")
-			.renameTo(new File("$buildDir/asciidoc/guide/index-docinfo.xml"))
-
-	}
-}
-
-task checkDocsLinks(type: JavaExec, dependsOn: [":documentation-toolchain:compileJava", pullDocs, shellReferenceDoc, moduleOptionsReferenceDoc]) {
+task checkDocsLinks(type: JavaExec, dependsOn: [":documentation-toolchain:compileJava", shellReferenceDoc, moduleOptionsReferenceDoc]) {
 	classpath = project(':documentation-toolchain').sourceSets.main.runtimeClasspath
 	main = 'org.springframework.xd.documentation.AsciidocLinkChecker'
 	args = [
-		"file:${pullDocs.cloneDestination}/guide/*.asciidoc"
+		"file:$docsDir/*.asciidoc"
 	]
 }
 
-apply plugin: org.asciidoctor.gradle.AsciidoctorPlugin
-
-task asciidoctorDocbook(type: AsciidoctorTask, dependsOn: [	transformDocs, shellReferenceDoc, moduleOptionsReferenceDoc] ) {
-	classpath = project.configurations.asciidoctor
-	sourceDocumentName = file("$buildDir/asciidoc/guide/index.adoc")
-	sourceDir = file("$buildDir/asciidoc/guide")
-	baseDir = sourceDir
-	outputDir = file("$buildDir/docbook")
-	backend = "docbook"
-	options = [
-		attributes: [
-			docinfo: '',
-			'compat-mode': '',
-			appversion: "$version"
-		]
-	]
 
-	doLast {
-		copy {
-			from "$buildDir/asciidoc/images"
-			into "$buildDir/docbook/images"
-		}
-	}
-}
+apply plugin: org.asciidoctor.gradle.AsciidoctorPlugin
 
-task asciidoctorHtml(type: AsciidoctorTask, dependsOn: [transformDocs, shellReferenceDoc, moduleOptionsReferenceDoc ]) {
-	group = 'Documentation'
-	description = 'Generate the HTML documentation using Asciidoctor'
-
-	classpath = project.configurations.asciidoctor
-	sourceDocumentName = file("$buildDir/asciidoc/guide/index.adoc")
-	sourceDir = file("$buildDir/asciidoc/guide")
-	baseDir = sourceDir
-	outputDir = file("$buildDir/html")
-	backend = "html5"
+asciidoctor {
+	sourceDir file("$docsDir")
+	sourceDocumentNames = files("$docsDir/index.asciidoc") // Change in >= 1.5.1
+	outputDir file("$buildDir/html")
+	backends = ['html5', 'docbook']
+	logDocuments = true
 	options = [
+		doctype: 'book',
 		attributes: [
 			docinfo: '',
 			toc2: '',
 			'compat-mode': '',
-			imagesdir: 'images/',
-			stylesdir: "$baseDir/stylesheets/",
+			imagesdir: '',
+			stylesdir: "stylesheets/",
 			stylesheet: 'golo.css',
 			appversion: "$version",
 			'source-highlighter': 'highlightjs'
 		]
 	]
-
-	doFirst {
-		copy {
-			from "$buildDir/asciidoc/guide/images"
-			into "$buildDir/html/images"
-		}
-		copy {
-			from "$buildDir/asciidoc/images"
-			into "$buildDir/html/images/images"
-		}
-		copy {
-			from "$buildDir/asciidoc/guide/stylesheets"
-			into "$buildDir/html/stylesheets"
-		}
-	}
 }
 
+asciidoctor.dependsOn ([checkDocsLinks, shellReferenceDoc, moduleOptionsReferenceDoc])
+
 apply plugin: DocbookReferencePlugin
 
 reference {
 	sourceFileName = 'index.xml'
-	sourceDir = file("$buildDir/docbook")
+	sourceDir = file("$buildDir/html")
```

Comment:
Yeah, saw that and thought about it.
I guess it doesn't hurt *that much*, especially given that accessing `thedocs/` will display `thedocs/index.html`, so only people aware of it would see it